src/commands/lighthouse/audit.ts

```diff
@@ -0,0 +1,28 @@
+import { flags as oclifFlags } from '@oclif/command'
+
+import { CustomCommand } from '../../oclif/CustomCommand'
+import auditUrl from '../../modules/lighthouse/auditUrl'
+
+export default class AuditUrl extends CustomCommand {
+  static description = 'Run lighthouse audit over a specific url'
+
+  static examples = ['vtex lighthouse audit my.url.com', 'vtex lh audit my.url.com']
+
+  static aliases = ['lh:audit']
+
+  static flags = {
+    ...CustomCommand.globalFlags,
+    json: oclifFlags.boolean({ char: 'j', description: 'Return the report as json on stdout', default: false }),
+  }
+
+  static args = [{ name: 'url' }]
```

Comment:
Is this required? If so I think there should be a `required: true` property

Reply:
Do you mean the `url` argument, right? Nice, I have missed that!